spec/requests/edit_user_spec.rb

```diff
@@ -116,27 +113,12 @@ def sign_in_as_a_valid_user(user)
     end
   end
 
-  context 'user adds a new mobile number but not mobile 2FA' do
-    before do
-      sign_in_as_a_valid_user(user)
-      patch_via_redirect '/users', id: @user, user: attrs_with_email_2fa
-    end
-
-    it 'deletes unconfirmed number after log out if not confirmed' do
```

Comment:
Was this removed because its a duplicate of another spec?  (could be seperate change too)